### PR TITLE
Feat/logs

### PR DESCRIPTION
**Pull request by xzdry**
improve `WarningArea` and `Logs` component：

- add display of `mergeState`
- optimisize the display of parameters
- add `eventType` field，the event column will be displayed using the event type first（eg: `onClick`）, and will only be set to the default value（eg: `Trigger Event`） if it is not available
- add resizable function to WarningArea
- optimize component jumps to UI_TREE, STATE, DATASOURCE

logs

### CODE REVIEW

File: packages/runtime/src/utils/runEventHandler.ts [hunk 1]
```diff
@@ -38,6 +39,7 @@ export const runEventHandler = (
       name: evaledHandler.method.name,
       parameters: evaledHandler.method.parameters,
       triggerId,
+      eventType: eventType || (handler as any)?.type,
```
Comment (tanbowensg) [inline packages/runtime/src/utils/runEventHandler.ts]: Don't use `any`. `handler` has its own type `EventHandlerSpec`.

File: packages/runtime/src/services/debugger/type.ts [hunk 2]
```diff
@@ -6,5 +6,5 @@ export type DebugLog = {
 };
 
 export type DebuggerHandler = {
-  onDebug: (currentState: Record<string, any>, debugLog: DebugLog) => void;
+  onDebug: (debugLog: DebugLog, currentState: Record<string, any>) => void;
```
Comment (tanbowensg) [inline packages/runtime/src/services/debugger/type.ts:9]: Why change the params order? Does it matters?

File: packages/runtime/src/utils/runEventHandler.ts (hunk 1 above)
Comment (xzdry) [reply packages/runtime/src/utils/runEventHandler.ts]: changed

File: packages/runtime/src/services/debugger/type.ts (hunk 2 above)
Comment (xzdry) [reply packages/runtime/src/services/debugger/type.ts:9]: The debuglog functions provided in the previous implementation were not aligned with onDebug, which resulted in the need to manually reorder them when using them   
before reordering
```ts
import { initSunmaoUI,printDebugInfo } from '@sunmao-ui/runtime';
  const ui = initSunmaoUI({
    // ...
    debugHandler:{
      onDebug:(state,log)=>{
        printDebugInfo(log,state);
      }
    }
  });
```
 after reordering
```ts
import { initSunmaoUI,printDebugInfo } from '@sunmao-ui/runtime';
  const ui = initSunmaoUI({
    // ...
    debugHandler:{
      onDebug: printDebugInfo
    }
  });
```
I think the impact is minimal, as this version is not yet released

**Review by tanbowensg — APPROVED** [review]

**Pull request merged**